spec/factories/rents.rb

```diff
@@ -6,6 +6,8 @@
     lastname
     email
     phone
+    disclaimer true
+    purpose "Handla möbler på IKEA"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.